Signal/test/Models/MesssagesBubblesSizeCalculatorTest.swift

```diff
@@ -48,70 +48,70 @@ class MesssagesBubblesSizeCalculatorTest: XCTestCase {
         let text: String? = ""
         let viewItem = self.viewItemForText(text)
         let actual = messageBubbleSize(for: viewItem)
-        XCTAssertEqual(42, actual.height)
+        XCTAssertEqual(36, actual.height)
```

Comment:
IIRC these values reflect a certain version of iOS, a certain device model, and certain dynamic text sizes.